### ISSUE TITLE
Compare pointer to char (build failed gcc 7.3.0)

### ISSUE THREAD

**Issue by mail6543210** [osdlyrics/osdlyrics#42]
On ubuntu:bionic with gcc 7.3.0, it complains about:
```
src/ol_app_info.c:463:21: error: comparison between pinter and zero character constant [-Werror=pointer-compare]
       if (*pathiter != '\0')
                     ^~
src/ol_app_info.c:463:11: note: did you mean to dereference the pointer?
       if (*pathiter != '\0')
           ^
```

**Comment by Corax26** [comment]
Fixed by #47. Out of curiosity, why do you compile with -Werror (presumably configuring with --with-werror)? Not that I'm blaming you for that, so far I had definitely ignored this warning :D

**Comment by mail6543210** [comment]
Maybe some Debian packaging scripts I copied from add that? I forgot.
